server/filters/convert-image-uri.js

```diff
@@ -0,0 +1,75 @@
+
+export default function convertImageUri(originalUri, requiredSize, useIiif, useIiifOrigin) {
+  const imageMap = {
+    wordpress: {
```

Comment:
We are working on a wordpress migration now - so I would say we're say to assume this will disappear.